apps/doji-frontend/src/pages/expert/[id]/index.tsx

```diff
@@ -0,0 +1,41 @@
+import RatingPanel from '@frontend/components/Review/RatingPanel'
+import { httpClient } from '@frontend/services'
+import { ExtendedNextPage } from '@frontend/type'
+import { IExpertInfoDTO } from '@libs/api'
+import { Avatar, CircularProgress, Stack, Typography } from '@mui/material'
+import { styled } from '@mui/system'
+import { useRouter } from 'next/router'
+
+import { useQuery } from 'react-query'
+
+const ExpertPage: ExtendedNextPage = () => {
+  //TODO This Entire Page
+  const router = useRouter()
+  const { id } = router.query
+  const { isError, error, isLoading, data } = useQuery<IExpertInfoDTO>(['/expert/info', id], () =>
+    httpClient.get('/expert/info/' + encodeURIComponent(id as string)).then((res) => res.data),
```

Comment:
wonder why we need `encodeURIComponent` since `id` is already alphabet

Reply:
ok